src/effects/VST3/VST3EffectsModule.cpp

```diff
@@ -0,0 +1,301 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file VST3EffectsModule.cpp
+
+  @author Vitaly Sverchinsky
+
+  @brief Part of Audacity VST3 module
+
+**********************************************************************/
+
+
+#include "VST3EffectsModule.h"
+
+#include <stdexcept>
+
+#include <wx/stdpaths.h>
+#include <wx/dir.h>
+#include <wx/log.h>
+#include <public.sdk/source/vst/hosting/module.h>
+#include <public.sdk/source/vst/hosting/plugprovider.h>
+#include <pluginterfaces/vst/ivstaudioprocessor.h>
+
+
+#include "../../ModuleManager.h"
```

Comment:
This line was introduced in the commit "Some basic loading functionality" and the source file is added to the build in the following commit "VST3 sources added to CMakeLists" and the build fails because the include path should remove "../../"

Do an interactive rebase to fix the include path in the earlier commit.  Then each commit in the sequence will build.